lib-extra/src/main/resources/com/diffplug/spotless/extra/groovy_eclipse_formatter/v4.13.0.lockfile

```diff
@@ -0,0 +1,19 @@
+# Spotless formatter based on Groovy-Eclipse version 3.5.0 (see https://github.com/groovy/groovy-eclipse/releases)
+com.diffplug.spotless:spotless-eclipse-groovy:3.5.0
+com.diffplug.spotless:spotless-eclipse-base:3.2.1
+com.google.code.findbugs:annotations:3.0.0
+com.google.code.findbugs:jsr305:3.0.0
+org.eclipse.platform:org.eclipse.core.commands:3.9.500
+org.eclipse.platform:org.eclipse.core.contenttype:3.7.400
+org.eclipse.platform:org.eclipse.core.jobs:3.10.500
+org.eclipse.platform:org.eclipse.core.resources:3.13.500
+org.eclipse.platform:org.eclipse.core.runtime:3.16.0
+org.eclipse.platform:org.eclipse.equinox.app:1.4.300
+org.eclipse.platform:org.eclipse.equinox.common:3.10.500
+org.eclipse.platform:org.eclipse.equinox.preferences:3.7.500
+org.eclipse.platform:org.eclipse.equinox.registry:3.8.500
+org.eclipse.platform:org.eclipse.jface.text:3.15.300
+org.eclipse.platform:org.eclipse.jface:3.17.0
+org.eclipse.platform:org.eclipse.osgi:3.15.0
+org.eclipse.platform:org.eclipse.swt:3.112.0
```

Comment:
I always prevented to get the SWT in, since I don't want platform dependencies.
That's why I [explicitly exclude it](https://github.com/diffplug/spotless/blob/master/_ext/eclipse-groovy/build.gradle#L38).
Please have a look where it comes in. It should not be needed. The `_ext` test have the purpose to call all used functionality to check whether dependencies are missing.

Reply:
I removed it, but didn't investigate where it came in from.

Reply:
I believe jface had it as a dependency, will review

Reply:
It was jface, but if we can live without it, that's fine
https://repo1.maven.org/maven2/org/eclipse/platform/org.eclipse.jface/3.17.0/org.eclipse.jface-3.17.0.pom

Reply:
Perhaps should do the same trick that @fvgh linked above to jface.  The swt dependency doesn't have anything in it, but its pom does link out to the platform-specific dependencies.  So keeping swt does no harm (as long as we don't resolve transitives, it will blow up if we do resolve transitives), but it also does no good either.

Reply:
If it's just an aggregate artifact, then the existing exclusion should handle the transitive dependencies - I like the simplicity / cleanliness of that approach.

Reply:
Sorry, I think I misunderstood - jface vs jface-text.
Sure, sounds good!